### ISSUE TITLE
node-independent means of visualization for specific voxels

### ISSUE THREAD

**Issue by orenshatz** [knossos-project/knossos#292]
Nodes are specifically dedicated to the annotation of neurites.
Do we want to have different "kinds" of trees for simply choosing certain voxels in the datasets?
Or another kind of entity which isn't a node, because it definitely isn't *necessarily* part of a tree?

**Comment by Optiligence** [comment]
Trees are only agglomerates (`things` in the annotation.xml) of nodes. 
Nodes are only representations of voxels with some additional properties like radius and connectivity (which actually isn’t a node property in the file)
I’d still use nodes, with properties indicating their purpose. 
In the end you want to iterate over everything you want to render and not over different collections.

**Comment by orenshatz** [comment]
The difficulty is that any *existing* code iterating on node would have to be adjusted to ignore nodes of non-neurite property.

**Comment by my-tien** [comment]
I guess such a tree would be in another skeletonizer, not in the "neurite" skeleton.

**Comment by orenshatz** [comment]
But if there is currently a code that traverses all nodes/trees, then this code is unaware of the fact that a node might not belong to a "neurite" skeleton. Because so far all nodes were part of an annotated neurite.

**Comment by my-tien** [comment]
No, what I mean is a new skeletonizer instance. It would be a "meta" skeletonizer with its own first tree and first node.

**Comment by orenshatz** [comment]
Okay, that's reasonable. The question is whether there is currently a hardcoded reference from node-related code to a singleton skeletonizer, or we can simply create as many meta skeletonizers as we wish.
